manifest.json

```diff
@@ -1,17 +1,17 @@
 {
 	"name": "Pi-hole",
-	"id": "pihole",
+	"id": "pi-hole",
```

Comment:
Changing the id name may not be a good idea. I think it will cause the app upgrade to fail.